crates/task-impls/src/builder.rs

```diff
@@ -73,7 +73,9 @@ impl<TYPES: NodeType, Ver: StaticVersionType> BuilderClient<TYPES, Ver> {
     /// If the URL is malformed.
     pub fn new(base_url: impl Into<Url>) -> Self {
         Self {
-            inner: Client::new(base_url.into().join("block_info").unwrap()),
+            inner: Client::builder(base_url.into().join("block_info").unwrap())
+                .set_timeout(Some(Duration::from_secs(2)))
```

Comment:
Don't we want it to be equal to `builder_timeout`?

Reply:
I was thinking the client shouldn't know about consensus parameters, and this would just be a general "a simple HTTP request should never take longer than 2s or something is wrong"

Reply:
I think it's fine because we have the `builder_timeout` logic in consensus anyways